operator/ccnp_event.go

```diff
@@ -44,7 +45,11 @@ var (
 // using CiliumNetworkPolicy itself, the entire implementation uses the methods
 // associcated with CiliumNetworkPolicy.
 func enableCCNPWatcher() error {
-	log.Info("Starting to garbage collect stale CiliumClusterwideNetworkPolicy status field entries...")
+	enableCNPStatusUpdates := kvstoreEnabled() && option.Config.K8sEventHandover && !option.Config.DisableCNPStatusUpdates
+	if enableCNPStatusUpdates {
+		log.Info("Starting a CCNP Status handover from etcd to k8s...")
```

Comment:
Upstream commit had this changed to `kvstore` rather than `etcd`. But this is a minor thing.

Reply:
:+1: Realistically the v1.7 is only likely to be deployed with etcd anyway so I think it's fine that only the upstream/v1.8 versions to say that, I won't bother to follow up on this.

Reply:
Ah sorry, I forgot to make the same change here.